grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -1016,7 +1009,7 @@ private void copyRelation(ResourceImpl main, ResourceImpl<?> other, RelationImpl
         //Gets the other resource index and replaces all occurrences of the other resource id with the main resource id
         //This allows us to find relations far more quickly.
         String newIndex = otherRelation.getIndex().replaceAll(other.getId().getValue(), main.getId().getValue());
-        RelationImpl foundRelation = getTxCache().getModifiedRelations().get(newIndex);
+        RelationImpl foundRelation = getTxCache().getCachedRelation(newIndex);
```

Comment:
Makes the code shorter. `getCachedRelation` was defined in the cache but this was bypassing it. Functionally it does the same thing.